GabijaBern/assignment1/Anagrams.java

```diff
@@ -0,0 +1,85 @@
+//Q1: Given two strings, determine if one is an anagram of the other.
+//Parts: 1)stringSensitive()
+//		 2)stringInsensitive()
+//		 3)sentences()
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Anagrams {
+
+	
+	//-----------------first part of the question------------------------------
+	public static boolean stringSensitive(String firstWord, String secondWord)
+	{
+		//we could first check if those words are the same length
+		if(firstWord.length()!=secondWord.length()) 

```

Comment:
Style: it often makes the code more readable if you put spaces around operators like in:

```java
  if (firstWord.length() != secondWord.length()) {
    return false;
  }
```